www/src/components/searchbar-body.js

```diff
@@ -0,0 +1,225 @@
+import React, { Component } from 'react'
+import { InstantSearch, Hits, SearchBox, Stats, RefinementList, PoweredBy } from 'react-instantsearch/dom'
```

Comment:
You can try importing from /es/ to see if bundle size is any different with native esmodules vs commonjs, cc @samouss